papers/anton_malakhov/composability.rst

```diff
@@ -225,26 +225,26 @@ Here is an example of running the benchmark program in five different modes:
 .. code-block:: sh
     :linenos:
 
-    python bench.py         # Default OpenMP mode
-    KMP_BLOCKTIME=0 OMP_NUM_THREADS=4 \
-        python bench.py     # Tunned OpenMP mode
-    python -m SMP bench.py  # OpenMP + SMP mode
+    python bench.py             # Default OpenMP mode
+    KMP_BLOCKTIME=0 OMP_NUM_THREADS=1 \
+        python bench.py         # Tunned OpenMP mode
+    python -m SMP -f 1 bench.py # OpenMP + SMP mode
     KMP_COMPOSABILITY=mode=exclusive \
-        python bench.py     # Composable OpenMP mode
-    python -m TBB bench.py  # Composable TBB mode
+        python bench.py         # Composable OpenMP mode
+    python -m TBB bench.py      # Composable TBB mode
 
 .. figure:: dask_static.png
 
    Execution times for balanced QR decomposition workload. :label:`sdask`
 
 Figure :ref:`sdask` shows performance results acquired on a 44-core (88-thread) machine with 128 GB memory. The results presented here were acquired with cpython v3.5.2; however, there is no significant performance difference with cpython v2.7.12.
-By default Dask will process a chunk in a separate thread so there will be 44 threads on the top level. Also each chunk will be computed in parallel with 44 OpenMP workers.
+By default Dask will process a chunk in a separate thread so there will be 44 threads on the top level (note that by default Dask will create a thread pool with 88 workers but only half of them will be really used since there are only 44 chunks). Also each chunk will be computed in parallel with 44 OpenMP workers.
```

Comment:
It works too long. I can try to reduce size of matrix and use 88 chunks but need to check if we still will have reasonable results.